packages/odyssey-react-mui/src/Checkbox.tsx

```diff
@@ -72,12 +65,46 @@ export type CheckboxProps = {
    * The value attribute of the Checkbox
    */
   value?: string;
-} & SeleniumProps;
+} & (
+  | {
+      /**
+       * Sets the checked state of the Checkbox
+       */
+      isChecked: boolean;
+      /**
+       * Determines whether the Checkbox is checked
+       * Should not be used if `isChecked` is used
+       */
+      isDefaultChecked?: never;
```

Comment:
Do we have to put `never` or should we just leave it out altogether?

Reply:
It looks like we need to leave it as `never`. If we leave it out, then the function signature for `Checkbox` is invalid.